### PR TITLE
feature/VDX-246/added module and openapi file for content manager

### PR DESCRIPTION
**Pull request by sksadjad**

### CODE REVIEW

File: ecmo/openapi/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+
+paths:
+  /contacts:
+    get:
+      tags:
+        - Contacts
+      summary: Get the contacts by given parameters
+      operationId: getContacts
+      parameters:
+        - name: filter
```
Comment (BtencateSphereon) [inline ecmo/openapi/openapi.yaml:41]: i don't think this should be query but body param, same for all other filters

File: ecmo/openapi/openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,1325 @@
...
+
+      responses:
+        200:
+          description: Successfully fetched parties.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/PartyRelationship'
```
Comment (nklomp) [inline ecmo/openapi/openapi.yaml:75]: Why does get parties return relationships, but getting a single Party returns a Party?

These should be symetrical

File: ecmo/openapi/openapi.yaml [hunk 3]
```diff
@@ -0,0 +1,1325 @@
...
+          description: Unauthorized.
+        403:
+          description: Forbidden.
+        404:
+          description: Party does not exist.
+      parameters:
+        - $ref: '#/components/parameters/partyId'
+    put:
+      tags:
+        - Contact
```
Comment (nklomp) [inline ecmo/openapi/openapi.yaml:164]: Wrong tag

**Review by nklomp — CHANGES_REQUESTED** [review]
See remarks.

File: ecmo/openapi/openapi.yaml (hunk 2 above)
Comment (sksadjad) [reply ecmo/openapi/openapi.yaml]: changed this to list of `Party`

File: ecmo/openapi/openapi.yaml (hunk 3 above)
Comment (sksadjad) [reply ecmo/openapi/openapi.yaml]: changed this tag

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**